locust/locustfile.py

```diff
@@ -0,0 +1,26 @@
+from locust import HttpUser, task, constant_pacing
```

Comment:
This to test the performance right?
If yes then can we add a comment at the top stating it for better clarification.

Reply:
the name of the file: locustfile.py is the indicator